[2025-01-06T17:01:08.459+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-06T17:01:08.495+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kaggle_dag.creating_schema_stg manual__2025-01-06T17:01:03.433436+00:00 [queued]>
[2025-01-06T17:01:08.516+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kaggle_dag.creating_schema_stg manual__2025-01-06T17:01:03.433436+00:00 [queued]>
[2025-01-06T17:01:08.518+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-06T17:01:08.550+0000] {taskinstance.py:2889} INFO - Executing <Task(PostgresOperator): creating_schema_stg> on 2025-01-06 17:01:03.433436+00:00
[2025-01-06T17:01:08.561+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=104) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-06T17:01:08.564+0000] {standard_task_runner.py:72} INFO - Started process 105 to run task
[2025-01-06T17:01:08.564+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'kaggle_dag', 'creating_schema_stg', 'manual__2025-01-06T17:01:03.433436+00:00', '--job-id', '810', '--raw', '--subdir', 'DAGS_FOLDER/kaggle_dag.py', '--cfg-path', '/tmp/tmpoum1a9fc']
[2025-01-06T17:01:08.567+0000] {standard_task_runner.py:105} INFO - Job 810: Subtask creating_schema_stg
[2025-01-06T17:01:08.659+0000] {task_command.py:467} INFO - Running <TaskInstance: kaggle_dag.creating_schema_stg manual__2025-01-06T17:01:03.433436+00:00 [running]> on host 97bbb29a7501
[2025-01-06T17:01:08.879+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='kaggle_dag' AIRFLOW_CTX_TASK_ID='creating_schema_stg' AIRFLOW_CTX_EXECUTION_DATE='2025-01-06T17:01:03.433436+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-06T17:01:03.433436+00:00'
[2025-01-06T17:01:08.881+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-06T17:01:08.910+0000] {sql.py:278} INFO - Executing: CREATE SCHEMA IF NOT EXISTS stg;

CREATE TABLE IF NOT EXISTS stg.adv (
daily_time_spent_on_site float,
age SMALLINT,
area_income float,
daily_internet_usage float,
ad_topic_line VARCHAR,
city VARCHAR,
male SMALLINT,
country VARCHAR,
timestamp timestamp,
clicked_on_ad SMALLINT
);
[2025-01-06T17:01:08.935+0000] {base.py:84} INFO - Retrieving connection 'pg_connect'
[2025-01-06T17:01:08.961+0000] {base.py:84} INFO - Retrieving connection 'pg_connect'
[2025-01-06T17:01:08.971+0000] {sql.py:553} INFO - Running statement: CREATE SCHEMA IF NOT EXISTS stg;

CREATE TABLE IF NOT EXISTS stg.adv (
daily_time_spent_on_site float,
age SMALLINT,
area_income float,
daily_internet_usage float,
ad_topic_line VARCHAR,
city VARCHAR,
male SMALLINT,
country VARCHAR,
timestamp timestamp,
clicked_on_ad SMALLINT
);, parameters: None
[2025-01-06T17:01:09.044+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-06T17:01:09.046+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=kaggle_dag, task_id=creating_schema_stg, run_id=manual__2025-01-06T17:01:03.433436+00:00, execution_date=20250106T170103, start_date=20250106T170108, end_date=20250106T170109
[2025-01-06T17:01:09.104+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-06T17:01:09.227+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-06T17:01:09.233+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
